### ISSUE TITLE
Installer-based Spyder does not start on Windows

### ISSUE THREAD

**Issue by giumas** [spyder-ide/windows-installer-assets#5]
Using the latest installer, Spyder does not start on Windows. It only shows the splash screen, then nothing else happens. (How to activate the console or debug logging?)

I suspect that it is related to https://github.com/spyder-ide/qtawesome/issues/167 because it works when I change the Untrusted Font Blocking setting (described in https://docs.microsoft.com/en-US/troubleshoot/windows-client/shell-experience/feature-to-block-untrusted-fonts). Unfortunately, the listed solutions in the Microsoft page cannot be applied on the target machines.

**Comment by ccordoba12** [comment]
Yeah, this issue doesn't belong here. @dalthviz, please move it to our main repo.
